_includes/manuals/1.9/7.1_novnc.md

```diff
@@ -30,3 +30,7 @@ For VNC only, encrypted connections are the default on new installations.  If yo
 * Try a telnet/netcat connection from the Foreman host to the hypervisor hostname/port
 * The penultimate argument of websockify.py is the listening port number, check if your web client host can telnet to it
 * If using Firefox, check the known issues above and set the config appropriately
+* *WebSock error: [object Event]* can be caused by a self signed certificate, where the certificate`s algorithm is too weak, e.g. SHA1. Debugging with e.g. the Firefox JavaScript Console will print a warning similar to "This site makes use of a SHA-1 Certificate; it's recommended you use certificates with signature algorithms that use hash functions stronger than SHA-1. [https://developer.mozilla.org/docs/Security/Weak_Signature_Algorithm]". To solve this issue, use SHA-2 algorithms instead.
+Check your current algorithm used for the SSL certificate with openssl:
+  $ openssl x509 -in /etc/ssl/certs/foreman.domain.tld.crt -text -noout |grep Algorithm
```

Comment:
I think these two lines need another blank line in between and more indentation to show up formatted like code.